tracee-ebpf/tracee/tracee.go

```diff
@@ -303,7 +303,13 @@ func New(cfg Config) (*Tracee, error) {
 		t.Close()
 		return nil, err
 	}
+	if t.eventsToTrace[DetectHookedSyscallsEventID] {
+		path := "/tmp/tracee/out/tracee.pid"
+		ptmx, _ := os.OpenFile(path, os.O_RDONLY, 444)
```

Comment:
please handle errors. 
also - we should close the file at some point, no? not sure how this will impact the ioctl. maybe `defer` will solve this case?